Cargo.toml

```diff
@@ -1,6 +1,10 @@
 [workspace]
-resolver = "2"
-members = [
-    "buildpacks/ruby",
-    "commons"
-]
+members = ["buildpacks/ruby", "commons"]
+
+[workspace.package]
+rust-version = "1.64"
+edition = "2021"
+license = "BSD-3-Clause"
+
+[workspace.dependencies]
+libherokubuildpack = "0.12"
```

Comment:
None of these entries are used (adding them to the root doesn't make them used, they need to be referenced from the individual crates too) - I presume they are leftover from when the feature was being added as it's own CNB?

If you want to switch the Ruby CNB + commons crates to sharing metadata via the root `Cargo.toml`, then that should be in its own PR :-)